cdc/sink/mq/manager/kafka_manager.go

```diff
@@ -112,9 +131,80 @@ func (m *kafkaTopicManager) tryUpdatePartitionsAndLogging(topic string, partitio
 	}
 }
 
-// CreateTopic creates a topic with the given name
-// and returns the number of partitions.
-func (m *kafkaTopicManager) CreateTopic(topicName string) (int32, error) {
+func (m *kafkaTopicManager) getMetadataOfTopics() ([]*sarama.TopicMetadata, error) {
+	var topicList []string
+
+	start := time.Now()
+	m.topics.Range(func(key, value any) bool {
+		topic := key.(string)
+		topicList = append(topicList, topic)
+
+		return true
+	})
+
+	topicMetaList, err := m.admin.DescribeTopics(topicList)
+	if err != nil {
+		log.Error(
+			"Kafka admin client describe topics failed",
+			zap.Error(err),
+			zap.Duration("duration", time.Since(start)),
+		)
+		return nil, err
+	}
+	log.Info(
+		"Kafka admin client describe topics success",
+		zap.Duration("duration", time.Since(start)),
+	)
+
+	return topicMetaList, nil
+}
+
+// WaitUntilTopicCreationDone is called after CreateTopic to make sure the topic
+// can be safely written to. The reason is that it may take several seconds after
+// CreateTopic returns success for all the brokers to become aware that the topics have been created.
+// See https://kafka.apache.org/23/javadoc/org/apache/kafka/clients/admin/AdminClient.html
+func (m *kafkaTopicManager) WaitUntilTopicCreationDone(topicName string) error {
```

Comment:
It doesn't look like it needs to be exposed?

Reply:
> Great work! It would be nice if we could add a new monitor for the number of new topics and latency! But that doesn't block this pr merge.

I will put the monitoring stuff in another PR. The following metrics will be included:
- The amount of Kafka topics.
- Topic creation latency.
- Topic describe latency.

Reply:
> It doesn't look like it needs to be exposed?

Yes, it doesn't need to be exposed. I have modified it, PTAL.